### PR TITLE
Updated JsDebuggerTest to be enzyme 3 compatible

### PR DESCRIPTION
**Pull request by jmkulwik**
https://codedotorg.atlassian.net/browse/STAR-185 : Update *labs tests to be compatible with Enzyme v3.

### CODE REVIEW

**Review by ajpal — APPROVED** [review]
lgtm, assuming tests still pass 😄

**Comment by jmkulwik** [comment]
Will wait to merge this PR until the upgrade to enzyme 3

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ describe('The JSDebugger component', () => {
   });
 
   it('initially has the height of 120px', () => {
-    expect(debugAreaEl().style.height).to.equal('120px');
+    expect(debugAreaEl().props().style.height).to.equal(120);
```
Comment (joshlory) [inline apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: This is changing what's tested: from checking if it's actually the right height, to verifying a prop is passed in. Intentional?

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js [hunk 2]
```diff
@@ -124,22 +126,26 @@ describe('The JSDebugger component', () => {
       });
 
       it('will expand the debugger by setting the height in the css', () => {
-        expect(debugAreaEl().style.height).to.equal('120px');
+        expect(debugAreaEl().props().style.height).to.equal(120);
```
Comment (joshlory) [inline apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: > by setting the height in the css

I'm not sure this is true anymore, how the test is now written.

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js [hunk 3]
```diff
@@ -268,6 +265,28 @@ describe('The JSDebugger component', () => {
   });
 });
 
+function moveMouse(xTarget, yTarget) {
```
Comment (joshlory) [inline apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: Worth de-duplicating this code against `VisualizationResizeBarTest.jsx`? I could see the shared code going in a `test/util/...` file.

https://github.com/code-dot-org/code-dot-org/blob/2a2c4207ac702abaa90af0eb6640a80d4edd3462/apps/test/unit/lib/ui/VisualizationResizeBarTest.jsx#L65-L85

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js [hunk 4]
```diff
@@ -268,6 +265,28 @@ describe('The JSDebugger component', () => {
...
+  const event = document.createEvent('MouseEvents');
+  event.initMouseEvent(
```
Comment (joshlory) [inline apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: Nit: [`initMouseEvent` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent). Can use `new MouseEvent(...)` instead ([example](https://github.com/code-dot-org/blockly/pull/178/files#diff-c4f5317d617d000d8e1e13a253d6086eR78)).

Comment (jmkulwik) [reply apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: Leaving this as-is for two reasons:
1. new MouseEvent isn't supported in phantom js yet
2. this test (prior to the Enzyme 3 update) is broken in ChromeHeadless (and therefore likely also broken in Chrome) due to the way the events in other places are set up.

Due to # 2, this only works in phantom js. I'll create a future item to make this cross-browser compatible. Not doing this work as part of this change.

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js (hunk 1 above)
Comment (jmkulwik) [reply apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: Will fix

File: apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js (hunk 2 above)
Comment (jmkulwik) [reply apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js]: will fix

**Review by joshlory — APPROVED** [review]
LGTM!

**Pull request merged**